cpp/src/io/orc/writer_impl.cu

```diff
@@ -55,10 +54,10 @@ using pinned_buffer = std::unique_ptr<T, decltype(&cudaFreeHost)>;
 constexpr orc::CompressionKind to_orc_compression(
     compression_type compression) {
   switch (compression) {
+    default:
```

Comment:
If the requested compression type is GZIP, BZIP2, BROTLI, XZ, etc., is it best to silently map that to snappy?  It may be better to throw an unsupported operation exception.

Reply:
Done.